core/src/main/java/com/klaytn/caver/wallet/keyring/KeyStore.java

```diff
@@ -0,0 +1,1160 @@
+
+/**
+ * Represents a KeyStore DTO(Data transfer Object) class according to KIP-3.
+ * For more details, please see below link.
+ * http://kips.klaytn.com/KIPs/kip-3
+ */
+public class KeyStore {
+    /**
+     * Keystore Format V4 version
+     */
+    public static final int KEY_VERSION_V3 = 3;
+
+    /**
+     * Keystore Format V3 version
+     */
+    public static final int KEY_VERSION_V4 = 4;
+
+    /**
+     * An address in KeyStore
+     */
+    private String address;
+
+    /**
+     * An crypto in KeyStore V3
+     */
+    private KeyStore.Crypto crypto;
+
+    /**
+     * An keyring in KeyStore V4
+     */
+    private List keyRing;
+
+    /**
+     * An ID in KeyStore
+     */
+    private String id;
+
+    /**
+     * A Version in KeyStore
+     */
+    private int version;
+
+    /**
+     * Creates a KeyStore instance.
+     */
+    public KeyStore() {
+    }
+
+    /**
+     * Setter function of keyRing.
+     * @param keyRing
+     */
+    @JsonSetter("keyring")
+    @JsonDeserialize(using = KeyStore.KeyRingDeserializer.class)
+    public void setKeyring(List keyRing) {
+        this.keyRing = keyRing;
+    }
+
+    /**
+     * Getter function of keyRing.
+     * @return List
+     */
+    public List getKeyring() {
+        return this.keyRing;
+    }
+
+    /**
+     * Getter function of address.
+     * @return String
+     */
+    public String getAddress() {
+        return address;
+    }
+
+    /**
+     * Setter function of address.
+     * @param address An address
+     */
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    /**
+     * Getter function of crypto.
+     * @return KeyStore.Crypto
+     */
+    public KeyStore.Crypto getCrypto() {
+        return crypto;
+    }
+
+    /**
+     * Setter function of crypto. for KeyStore Format V3.
+     * @param crypto KeyStore.Crypto.
+     */
+    @JsonSetter("crypto")
+    public void setCrypto(KeyStore.Crypto crypto) {
+        this.crypto = crypto;
+    }
+
+    /**
+     * Setter function of Crypto. for KeyStore Format V1.
+     * @param crypto KeyStore.Crypto
+     */
+    @JsonSetter("Crypto")  // older wallet files may have this attribute name
+    public void setCryptoV1(KeyStore.Crypto crypto) {
+        setCrypto(crypto);
+    }
+
+    /**
+     * Getter function for id.
+     * @return String
+     */
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * Setter function for id.
+     * @param id An id in KeyStore
+     */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * Getter function for version.
+     * @return integer
+     */
+    public int getVersion() {
+        return version;
+    }
+
+    /**
+     * Setter function for version
+     * @param version version in KeyStore
+     */
+    public void setVersion(int version) {
+        this.version = version;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
It looks like comparing `keyRing` is missing here.

Reply:
I'll modify it.